src/pages/public_registration/RegistrationSummary.tsx

```diff
@@ -18,11 +18,11 @@ export const RegistrationSummary = ({ id }: RegistrationSummaryProps) => {
 
   const containerQuery = useQuery({
     queryKey: ['container', identifier],
-    queryFn: () => fetchResults({ id: identifier, results: 1 }),
+    queryFn: () => fetchRegistration(identifier),
     meta: { errorMessage: t('feedback.error.search') },
   });
 
-  const container = containerQuery && containerQuery.data?.hits.length === 1 ? containerQuery.data.hits[0] : null;
+  const container = containerQuery.data ? containerQuery.data : null;
```

Comment:
blir ikke dette det samme?
```suggestion
  const container = containerQuery.data;
```